src/System.Net.Mail/src/System.Net.Mail.csproj

```diff
@@ -6,6 +6,7 @@
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <IsPartialFacadeAssembly Condition="'$(TargetGroup)'=='net463'">true</IsPartialFacadeAssembly>
     <NuGetTargetMoniker Condition="'$(TargetGroup)' == ''">.NETStandard,Version=v1.7</NuGetTargetMoniker>
+    <DefineConstants Condition="'$(TargetGroup)' == ''">NETSTANDARD1_7</DefineConstants>
```

Comment:
@weshaggard @ericstj Is there a better way to achieve this? (some already defined constant)
If not, can we otherwise move the constant definition to the root proj file?